### ISSUE TITLE
Accessibility Issues with Navigation bar

### ISSUE THREAD

**Issue by phucduong86** [shiffman/shiffman.net#69]
1. Keyboard User cannot activate nav sidebar using keyboard
2. Keyboard focus is lost for sighted user. Please see screenshot - I use screen reader with the focus addon to show where the keyboard focus currently is, the links are announce and focus is active but it os off the screen so the sighted user won't know which feature they are activating.

Recommendation:
- Implement Expand/Collapse feature on sidebar , activated on click instead of on hover
- Links within the sidebar should only be activated when the sidebar is in "expanded" state (prevent focus lost)
![shiffman-navbar](https://user-images.githubusercontent.com/21692768/47744799-9cc21d00-dc58-11e8-853a-d374be317c40.png)

**Comment by phucduong86** [comment]
The sidebar is inconsistent on different pages.
**On the home page**
1. After the sidebar is expanded it is directly overlayed on top of content, makes it hard to read.
![shiffman-navbar2](https://user-images.githubusercontent.com/21692768/48000702-53e4eb00-e0d4-11e8-8479-649b306778e3.png)
Looks much better on the other page (https://shiffman.net/books/) for example
![shiffman-book](https://user-images.githubusercontent.com/21692768/48000825-b938dc00-e0d4-11e8-9ab8-412e26956a5a.png)

2. Common issue on both pages though - to collapse the sidebar is not easy, user needs to touch the area outside of the nav sidebar div, which is a very narrow area along the right edge of the screen.

**Comment by phucduong86** [comment]
Resolved by PR #98
